docs/docs/adding-comments.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Adding comments
+---
+
+If you're using Gatsby to run a blog and you've started adding some content to it, the next thing to think about is how to increase engagement among your visitors. A great way to do that is to allow them to ask questions and express their views on what you've written. This will make your blog seem much more lively to anyone visiting it.
+
+There are many options out there for adding comment functionality, several of them specifically targeted at static sites. While this list is by no means exhaustive, it does serve as a good starting point to illustrate what's available:
+
+- [Disqus](https://disqus.com)
+- [Facebook comments](https://www.npmjs.com/package/react-facebook)
+- [Staticman](https://staticman.net)
+- [JustComments](https://just-comments.com)
+- [TalkYard](https://www.talkyard.io)
+- [Gitalk](https://gitalk.github.io)
+
+In this guide, we'll show you how to implement Disqus on your blog as it has a number of great features.
+
+- It is low maintenance, meaning [moderating your comments and maintaining your forum](https://help.disqus.com/moderation/moderating-101) is easy.
+- It provides official [React support](https://github.com/disqus/disqus-react).
+- It offers a [generous free tier](https://disqus.com/pricing).
+- It [seems to be by far the most widely used service](https://www.datanyze.com/market-share/comment-systems/disqus-market-share).
+- It’s easy to comment: Disqus has a large existing user base and the onboarding experience for new users is fast. You can register with your Google, Facebook or Twitter account and easily share your review about the post through those channels.
+- Its commenting interface has a distinct but unobtrusive look that many users will instantly recognize and trust.
+- All Disqus components are lazy-loaded, meaning they won't negatively impact the load time of your posts.
```

Comment:
We should note the privacy implications of adding Disqus to your site, and mention that you can potentially opt out of data sharing and ads in the settings (I haven't checked recently as I ditched Disqus for this reason a few years ago). 
- https://help.disqus.com/terms-and-policies/how-to-edit-your-data-sharing-settings
- https://replyable.com/2017/03/disqus-is-your-data-worth-trading-for-convenience/

Reply:
Are you suggesting we link those two articles in the text? Also, I'm not sure what kind of wording you're thinking of.

FWIW, I followed the link to the Disqus privacy settings in the second article intending to opt out of their user profiling and tracking. I was greeted by a page informing me that Disqus had detected the  _Do Not Track_ setting of my browser and that I would have to disable this setting first before I could _opt in_ to their ad training program.

![Screenshot 2019-04-08 at 20 37 29](https://user-images.githubusercontent.com/30958850/55751822-848fce80-5a3e-11e9-84ba-64c44e0dd8e8.png)

Reply:
A Disqus privacy settings doc would be good to include (not sure if there's a better one out there somewhere, but I did find a post on [GDPR compliance](https://blog.disqus.com/update-on-privacy-and-gdpr-compliance)). The intent of mentioning this would be to communicate to Gatsby site owners what they're responsible for in terms of a privacy policy (e.g. any related plugin embed settings) or if it is left up to end users with Do Not Track. 

I think it's fine to keep the privacy information in the post brief and link off to Disqus' own material so it stays up to date.

Reply:
I added two paragraphs on privacy concerns regarding Disqus.

Reply:
You could also mention alternatives:
https://victorzhou.com/blog/replacing-disqus/

Reply:
@LekoArts The guide already starts out with a list of alternatives. Commento seems like a good addition to that list.